### ISSUE TITLE
Demo on main page isn't working

### ISSUE THREAD

**Issue by sashkanz** [wojodesign/simplecart-js#492]
It looks like your demo shopping cart on http://simplecartjs.org/ isn't working...
"Uncaught ReferenceError: simpleCart is not defined"

**Comment by mudnug** [comment]
I agree that the demo doesn't work and wanted to upvote this, but noticed issue 456 seems similar.
